connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -18,14 +18,15 @@ package org.apache.spark.sql.connect.config
 
 import org.apache.spark.internal.config.ConfigBuilder
 import org.apache.spark.network.util.ByteUnit
+import org.apache.spark.sql.connect.common.config.ConnectCommon
 
 private[spark] object Connect {
 
   val CONNECT_GRPC_BINDING_PORT =
     ConfigBuilder("spark.connect.grpc.binding.port")
       .version("3.4.0")
       .intConf
-      .createWithDefault(15002)
+      .createWithDefault(ConnectCommon.CONNECT_GRPC_BINDING_PORT)
```

Comment:
Is there a good way to make the config builder part of the externally dependable interface?

Reply:
`ConfigBuilder` is part of spark core so we're unable to use it in common/client without the extra dep (unless we pull out the whole of `org.apache.spark.internal.config`). 

Seeing that the user can build the client on the go with complete customizability of host/port, it lets us get away without needing full support of configs via environment variables/config files in this case since:
- In a REPL environment, a new instance of the client can be made with an updated host/port. 
- In the case of applications, the client is supposed to be lightweight and minimal and hence rebuilding the package should be a trivial step.

WDYT?

Reply:
This matches with Python client side that Python client cannot share any configs from Spark thus it set the port by it own way.

From this perspective though that this port arg can stay within `client` but not in `common` because server will use the config and `client` is the only one use it. 

We can stay on this way for now.